master/buildbot/worker/latent.py

```diff
@@ -249,7 +250,11 @@ def insubstantiate(self, fast=False):
         try:
             yield d
         except Exception as e:
-            log.err(e, "while insubstanciating")
+            # The case of failure for insubstanciation is bad as we have a left-over costing ressource
+            # There is not much thing to do here generically, so we must put the problem of stop_instance
+            # reliability to the backend driver
+            log.err(e, "while insubstantiating")
```

Comment:
yeah. I think we should change the naming. as a non-native this does not make much sense, and I keep making typo.

@seankelly I dont know if as native it does mean something for you.